code/modules/mod/mod.dm

```diff
@@ -0,0 +1,549 @@
+		return
+
+/obj/item/clothing/head/helmet/space/mod
+	name = "MOD helmet"
+	desc = "A helmet for a MODsuit."
+	icon = 'icons/obj/mod.dmi'
+	icon_state = "helmet"
+	worn_icon = 'icons/mob/mod.dmi'
+	armor = list(MELEE = 0, BULLET = 0, LASER = 0, ENERGY = 0, BOMB = 0, BIO = 100, RAD = 0, FIRE = 25, ACID = 25, WOUND = 10)
+	body_parts_covered = HEAD
+	heat_protection = HEAD
+	cold_protection = HEAD
+	max_heat_protection_temperature = SPACE_SUIT_MAX_TEMP_PROTECT
+	min_cold_protection_temperature = SPACE_SUIT_MIN_TEMP_PROTECT
+	clothing_flags = THICKMATERIAL
+	resistance_flags = NONE
+	flash_protect = FLASH_PROTECTION_NONE
+	clothing_flags = SNUG_FIT
+	flags_inv = HIDEFACIALHAIR
+	flags_cover = HEADCOVERSMOUTH
+	visor_flags = THICKMATERIAL|STOPSPRESSUREDAMAGE
+	visor_flags_inv = HIDEMASK|HIDEEARS|HIDEEYES|HIDEFACE|HIDEHAIR
+	visor_flags_cover = HEADCOVERSEYES|PEPPERPROOF
+	alternate_worn_layer = NECK_LAYER
+	var/obj/item/mod/control/mod
+
+/obj/item/clothing/head/helmet/space/mod/Destroy()
+	..()
+	if(mod)
+		mod.helmet = null
+		QDEL_NULL(mod)
+
+/obj/item/clothing/suit/armor/mod
+	name = "MOD chestplate"
+	desc = "A chestplate for a MODsuit."
+	icon = 'icons/obj/mod.dmi'
+	icon_state = "chestplate"
+	worn_icon = 'icons/mob/mod.dmi'
+	blood_overlay_type = "armor"
+	armor = list(MELEE = 0, BULLET = 0, LASER = 0, ENERGY = 0, BOMB = 0, BIO = 100, RAD = 0, FIRE = 25, ACID = 25, WOUND = 10)
+	body_parts_covered = CHEST|GROIN
+	heat_protection = CHEST|GROIN
+	cold_protection = CHEST|GROIN
+	max_heat_protection_temperature = SPACE_SUIT_MAX_TEMP_PROTECT
+	min_cold_protection_temperature = SPACE_SUIT_MIN_TEMP_PROTECT
+	clothing_flags = THICKMATERIAL
+	visor_flags = STOPSPRESSUREDAMAGE
+	visor_flags_inv = HIDEJUMPSUIT
+	allowed = list(/obj/item/flashlight, /obj/item/tank/internals)
+	resistance_flags = NONE
+	var/obj/item/mod/control/mod
+
+/obj/item/clothing/suit/armor/mod/Destroy()
+	..()
+	if(mod)
+		mod.chestplate = null
+		QDEL_NULL(mod)
+
+/obj/item/clothing/gloves/mod
+	name = "MOD gauntlets"
+	desc = "A pair of gauntlets for a MODsuit."
+	icon = 'icons/obj/mod.dmi'
+	icon_state = "gauntlets"
+	worn_icon = 'icons/mob/mod.dmi'
+	armor = list(MELEE = 0, BULLET = 0, LASER = 0, ENERGY = 0, BOMB = 0, BIO = 100, RAD = 0, FIRE = 25, ACID = 25, WOUND = 10)
+	body_parts_covered = HANDS|ARMS
+	heat_protection = HANDS|ARMS
+	cold_protection = HANDS|ARMS
+	max_heat_protection_temperature = SPACE_SUIT_MAX_TEMP_PROTECT
+	min_cold_protection_temperature = SPACE_SUIT_MIN_TEMP_PROTECT
+	clothing_flags = THICKMATERIAL
+	resistance_flags = NONE
+	var/obj/item/mod/control/mod
+	var/obj/item/clothing/overslot
+
+/obj/item/clothing/gloves/mod/Destroy()
+	..()
+	if(mod)
+		mod.gauntlets = null
+		QDEL_NULL(mod)
+
+/obj/item/clothing/gloves/mod/dropped(mob/user)
+	. = ..()
+	if(!overslot)
+		return
+	user.dropItemToGround(overslot, TRUE, TRUE)
+	user.equip_to_slot_if_possible(overslot, overslot.slot_flags, FALSE, TRUE)
+	overslot = null
+
+/obj/item/clothing/shoes/mod
```

Comment:
should digitigrade lizards be allowed to wear these?

Reply:
iudk